apps/showcase/src/app/app.component.ts

```diff
@@ -43,7 +45,14 @@ export class AppComponent implements OnDestroy {
 
   private subscriptions = new Subscription();
 
-  constructor(router: Router, private offcanvasService: NgbOffcanvas) {
+  constructor(
+    router: Router,
+    localizationDevtoolsContextualizationService: LocalizationDevtoolsContextualizationService,
+    configurationDevtoolsContextualizationService: ConfigurationDevtoolsContextualizationService,
+    private offcanvasService: NgbOffcanvas
+  ) {
+    localizationDevtoolsContextualizationService.activate();
```

Comment:
you could add some similar doc as for this one
https://github.com/AmadeusITGroup/otter/blob/main/docs/localization/LOCALIZATION.md#enable-chrome-extension-debugging

Reply:
First I would like we discuss about how we expose this api 
I did it this way to test it 
but I'm not sure it will be the final way to do it